examples/server/api_like_OAI.py

```diff
@@ -183,14 +185,16 @@ def generate():
                     decoded_line = line.decode('utf-8')
                     resData = make_resData_stream(json.loads(decoded_line[6:]), chat=True, time_now=time_now)
                     yield 'data: {}\n'.format(json.dumps(resData))
```

Comment:
There need to be TWO `\n` to ensure that all chat clients can understand the streamed data chunks. 

Should read `yield 'data: {}\n\n'.format(json.dumps(resData))`